databases/models/UserTopics.js

```diff
@@ -20,6 +20,8 @@ module.exports = (sequelize, DataTypes) => {
       },
       user_id: {type: DataTypes.STRING, allowNull: false},
       topic_id: {type: DataTypes.BIGINT, allowNull: false},
+      notify_user: {type: DataTypes.BOOLEAN, defaultValue: false, allowNull: false},
+      is_anonymous: {type: DataTypes.BOOLEAN, defaultValue: false, allowNull: false},
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding new fields `notify_user` and `is_anonymous` to the model seems fine. However, please make sure that these changes are reflected in any related queries or operations that interact with this model. Also, consider adding comments to describe what these fields are used for, which will improve maintainability.
```diff
+      notify_user: {type: DataTypes.BOOLEAN, defaultValue: false, allowNull: false},
+      is_anonymous: {type: DataTypes.BOOLEAN, defaultValue: false, allowNull: false},
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->